client/packages/system/src/Patient/hooks/usePatientModalStore.ts

```diff
@@ -0,0 +1,31 @@
+import { zustand } from '@openmsupply-client/common';
+import { PatientModal } from '../DetailView';
+
+interface PatientModalState {
+  current: PatientModal | undefined;
+  documentName: string | undefined;
+  documentType: string | undefined;
```

Comment:
Isn't this more common:

```suggestion
  documentType?: string;
```

Reply:
heh. I had `| null` initially and then changed to `undefined` due to a downstream usage. Didn't think it through..